.circleci/config.yml

```diff
@@ -192,7 +192,7 @@ jobs:
       - run:
           name: Build and run tests
           no_output_timeout: 35m
-          command: cd testing && RUST_MIN_STACK=8388608 cargo test --release -- --nocapture
+          command: cd testing && RUST_MIN_STACK=8388608 cargo test --release -- --nocapture --skip metrics && RUST_MIN_STACK=8388608 cargo test --release metrics -- --test-threads 1
```

Comment:
nit: it would be more readable to run 2 separate commands here (still with the same build, though); also, the metrics tests probably don't need a stack this large, but it might still be required due to how the node spawns the `tokio` executor :thinking: